addons/gdUnit4/src/core/parse/GdFunctionDescriptor.gd

```diff
@@ -146,7 +146,7 @@ static func _extract_args(method_descriptor :Dictionary) -> Array[GdFunctionArgu
 		var arg :Dictionary = arguments.pop_back()
 		var arg_name := _argument_name(arg)
 		var arg_type := _argument_type(arg)
-		var arg_default := GdFunctionArgument.UNDEFINED
+		var arg_default = GdFunctionArgument.UNDEFINED as Variant
```

Comment:
This solves this error:
```
SCRIPT ERROR: Parse Error: Could not infer the type of the variable "arg_default" because the initial value is a variant. Use explicit "Variant" type if this is intended.
```

Reply:
Can you give me a hint when you see the error?
When I open the script, no error is displayed.
The constant `GdFunctionArgument.UNDEFINED` has type `Variant`
and the type assignment should be ok `var arg_default := GdFunctionArgument.UNDEFINED`

Reply:
As workaround `var arg_default: Variant = GdFunctionArgument.UNDEFINED` works too